test/int.jl

```diff
@@ -176,6 +176,19 @@ end
 @test widemul(false, 3) == 0
 @test widemul(3, true) == widemul(true, 3) == 3
 
+i=Int64(2)^63-1
```

Comment:
this might be better as a `let i = Int64(2)^63, k = widemul(i,i)`
block since these are common variable names